project/eScavenge/StringConst.py

```diff
@@ -23,10 +23,10 @@
 edit_landmark_fail = "Landmark Not Edited Successfully"
 edit_landmark_order_success = "Landmark Order Edited Successfully"
 edit_landmark_order_fail = "Landmark Order Not Edited Successfully"
+no_team_logged_in = "There is no Team Logged In"
```

Comment:
Only thing I can see that I can comment on is just that this instance of no_team_logged_in isn't used anywhere, but I think that came from an old commit of mine